### PR TITLE
Fix task links; other task requirements

### PR DESCRIPTION
**Pull request by Razzmatazzz**

### CODE REVIEW

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

File: src/tarkov-data-manager/data/changed_quests.json [hunk 1]
```diff
@@ -635,5 +652,92 @@
...
+                    "name": "Scout",
+                    "status": [
+                        "complete"
+                    ]
+                }
+            ]
+        }
+    },
+    "6089732b59b92115597ad789": {
+        "name": "Surplus GOods",
```
Comment (Shebuka) [inline src/tarkov-data-manager/data/changed_quests.json:692]: Typ**O**?

**Comment by Shebuka** [comment]
Lots of changes to quests, are they all from parsing the Wiki?

**Comment by Razzmatazzz** [comment]
> Lots of changes to quests, are they all from parsing the Wiki?

Manually parsing, yes.

File: src/tarkov-data-manager/data/changed_quests.json (hunk 1 above)
Comment (Razzmatazzz) [reply src/tarkov-data-manager/data/changed_quests.json:1014]: Yeah, but the names aren't critical. They're only used to make it human-readable.

**Comment by Razzmatazzz** [comment]
.deploy

File: src/tarkov-data-manager/jobs/update-quests.js [hunk 2]
```diff
@@ -235,6 +244,26 @@ class UpdateQuestsJob extends DataJob {
...
+                addPreviousRequirements(neededSet, req.task, req.status.length === 1 && req.status[0] === 'complete');
+            }
+        };
+        addPreviousRequirements(neededForKappa, '5c51aac186f77432ea65c552', true);
+        addPreviousRequirements(neededForLightkeeper, '625d7005a4eb80027c4f2e09', true);
+        for (const task of quests.Task) {
+            if (neededForKappa.has(task.id)) {
+                task.kappaRequired = true;
+                task.lightkeeperRequired = true;
+            }
```
Comment (Shebuka) [inline src/tarkov-data-manager/jobs/update-quests.js:264]:
```suggestion
            if (neededForKappa.has(task.id)) {
                task.kappaRequired = true;
            }
            if (neededForLightkeeper.has(task.id)) {
                task.lightkeeperRequired = true;
            }
```

**Review by Shebuka — CHANGES_REQUESTED** [review]

File: src/tarkov-data-manager/jobs/update-quests.js (hunk 2 above)
Comment (Razzmatazzz) [reply src/tarkov-data-manager/jobs/update-quests.js]: Good catch!

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Review by GrantBirki — APPROVED** [review]

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
,deploy

**Review by GrantBirki — APPROVED** [review]

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Review by GrantBirki — APPROVED** [review]

**Review by Shebuka — APPROVED** [review]

**Pull request merged**